cpp/benchmarks/bench_simulation.cpp

```diff
@@ -0,0 +1,56 @@
+/* 
+* Copyright (C) 2020-2021 German Aerospace Center (DLR-SC)
+*
+* Authors: Rene Schmieding
+*
+* Contact: Martin J. Kuehn <Martin.Kuehn@DLR.de>
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*     http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+#include "bench_core.h"
+#include "bench_model.h"
+

```

Comment:
I have some concerns about your files
- bench_core is a very vague name. The file contains code that is specific to the integrator and the simulation.
- bench_model is misleading, the model isn't benchmarked, just initialized. 

I think bench_core can be reduced to just the config file stuff and renamed accordingly. bench_model should be renamed to reflect its purpose better.